### ISSUE TITLE
Error in making font - file not created before dumping

### ISSUE THREAD

**Issue by NightFurySL2001** [CyanoHao/Resource-Han-Rounded#5]
When trying to make the font file with my own Source Han Sans font, I put the .otd file in the src folder and run `round.py`. After running for a long time the following message occur:
```
Traceback (most recent call last):
  File "round.py", line 660, in <module>
    RoundFont(region, weight)
  File "round.py", line 653, in RoundFont
    with open("out/ResourceHanRounded{}-{}.otd".format(region, weight), 'w') as outFile:
FileNotFoundError: [Errno 2] No such file or directory: 'out/ResourceHanRoundedSC-Regular.otd'
```
It seems to not create a file before dumping the JSON. Is this expected behaviour?

**Comment by CyanoHao** [comment]
Run `mkdir -p out/` first.

**Comment by NightFurySL2001** [comment]
Thanks.
Here's another error:
```
Traceback (most recent call last):
  File "round.py", line 660, in <module>
    RoundFont(region, weight)
  File "round.py", line 654, in RoundFont
    outFile.write(outStr)
UnicodeEncodeError: 'gbk' codec can't encode character '\xa9' in position 3209: illegal multibyte sequence
```
Can you help on this one?

**Comment by CyanoHao** [comment]
Should be fixed in commit#17128f2. It’s now UTF-8 encoded.

**Comment by NightFurySL2001** [comment]
Ok, its fixed. Thanks for helping.
